tests/test_chat_include_orchestration_trace_http_honesty.py

```
@@ -53,6 +53,97 @@ def _server():
+        orchestrator,
+        port=0,
+        security=SecurityConfig(bearer_verifier=verifier),
+    )
+    thread = threading.Thread(target=server.serve_forever, daemon=True)
+    thread.start()
+    return server, thread, server.server_address[1], orchestrator
+
+
+def test_trace_requires_a_verified_trace_purpose() -> None:
+    server, thread, port, _orchestrator = _server_with_verifier(
+        lambda token, scope: token == "inference_only" and scope == "inference"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "token"

(S105)